cves/kernel/CVE-2013-3223.yml

```diff
@@ -75,7 +92,9 @@ bugs_instructions: |
     * Mentioned in mailing list discussions
     * References from NVD entry
     * Various other places
+# No bug IDs
 bugs: []
```

Comment:
I feel like there should be some sort of bug ID as this is an issue within the software meaning that there should be a bug. I would suggest to dig deeper for any bug id mentioned in any of the sources.